auth_u2f/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.png
```

Comment:
Maybe we can use the Readme By Fragments
* https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
* https://github.com/OCA/maintainer-tools/blob/master/README.md

Reply:
Sounds like a good idea.